.github/workflows/osx.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         run: |
           set -ex
           brew update
-          brew install freetype fontconfig qt5 harfbuzz boost pkg-config
+          brew install freetype fontconfig glew glfw3 glm qt5 harfbuzz boost pkg-config
```

Comment:
Why are these 3 new deps needed? (did I forget that somewhere? it shouldn't be needed anymore)

Reply:
I figured to add it because it's in README.md

Reply:
Removed instances of them from yml and readme